MANIFEST.in

```diff
@@ -1,3 +1,15 @@
 include logo.txt
 include scripts/*
 include tests/*
+include license
+include CHANGELOG.rst
+include README.rst
+graft docs
+graft news
```

Comment:
This is only used for drafting during development and is rolled into CHANGES as part of the release process.